packages/runtime/container-runtime/src/summaryManagerUtils.ts

```diff
@@ -0,0 +1,200 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IEvent } from "@fluidframework/common-definitions";
+import { assert, TypedEventEmitter } from "@fluidframework/common-utils";
+import { IQuorum, ISequencedClient } from "@fluidframework/protocol-definitions";
+
+export const summarizerClientType = "summarizer";
```

Comment:
nit. can this file just be called OrderedClients, as utils file just end up filled with junk